tolicodes.com-frontend/src/components/NavBar/SNavBar.ts

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+
+export const SNavBar = styled.nav`
+  ul {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    li {
+      margin: 0 2.5px;
+      padding: 15px;
+      background-color: var(--ui-05);
+
+      a {
```

Comment:
Make these into their own components

Reply:
Why? If we put everything in their own components, we're gonna have million of components that are used only in their parents components.

Reply:
that's ok :) Let's keep things very modular, because this will be reused elsewhere. Trust me ;)